website/content/en/wgs/infra-lifecycle/glossary/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Glossary
+description: "Defines key terms used in the Infrastructure Lifecycle Working Group’s writings."
+---
+
+See also: [Cloud Native Glossary](https://glossary.cncf.io/)
+
+If you wish to refer to these definitions outside the scope of the working group’s papers, please note that they have been written in the context of both the CNCF and infrastructure management.
+
+## Infrastructure Lifecycle
+The continuous process of designing, deploying, operating, updating, and decommissioning infrastructure resources. This lifecycle covers stages from provisioning to maintenance and eventual retirement, with a focus on resilience, security, sustainability, and automation.
+
+## Infrastructure as Code (IaC)
+A practice where infrastructure is managed and provisioned through machine-readable configuration files, enabling consistency and scalability. Allowing infrastructure to be treated as code, enabling version control, testing, and automation of infrastructure management.
+
+## State
+In infrastructure management, state refers to the current configuration, status, and condition of infrastructure resources at any given point in time. It captures details like resource attributes, dependencies, and operational status, often stored in a state file.
```

Comment:
It feels like this needs to be more clear between:

- Desired state (what the user wrote down)
- Actual state (as described below in drift section)
- Stored state (eg: a Terraform state file, which might not be either 😉)

Reply:
I've removed the state definitions, we can work it out later in detail.